corehq/motech/requests.py

```diff
@@ -48,12 +53,33 @@ class Requests(object):
     To maintain a session of authenticated non-API requests, use
     Requests as a context manager.
     """
-    def __init__(self, domain_name, base_url, username, password, verify=True):
+
+    def __init__(self, domain_name, base_url, username, password,
+                 verify=True, notify_email=None):
+        """
+        Initialise instance
+
+        :param domain_name: Domain to store logs under
+        :param base_url: Remote API base URL
+        :param username: Remote API username
+        :param password: Remote API plaintext password
+        :param verify: Verify SSL certificate?
+        :param notify_email: A list, or a comma- or space-separated
+            string of email addresses to notify of errors.
```

Comment:
Why is this so flexible? Would it be easier to change `notify_email` to a `StringListProperty` and always pass in a list and error on any other inputs?